staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -330,6 +330,12 @@ message LinuxPodSandboxConfig {
     LinuxSandboxSecurityContext security_context = 2;
     // Sysctls holds linux sysctls config for the sandbox.
     map<string, string> sysctls = 3;
+
+    // overhead represents the overheads associated with this sandbox
+    LinuxContainerResources overhead = 4;
+
+    // sandbox_resources represents the sum of container resources for this sandbox
+    LinuxContainerResources sandbox_resources = 5;
```

Comment:
Maybe declare them as `optional`? So that it is easier for the CRI daemons to figure out if they are set.